packages/react-components/react-avatar/src/components/AvatarGroupItem/useAvatarGroupItem.ts

```diff
@@ -38,7 +38,7 @@ export const useAvatarGroupItem_unstable = (
       defaultProps: {
         style,
         className,
-        as: groupIsOverflow ? 'li' : 'div',
+        as: 'div',
```

Comment:
Is `as` needed anymore? Looks like the root slot can only be a div with the other changes in this PR.